app/views/questionnaire.py

```diff
@@ -41,16 +42,16 @@
 
 
 @questionnaire_blueprint.before_request
-@login_required
```

Comment:
Problem is I have end points which you are no longer are signed in for, the before_request will naturally fire for everything requests, if you have to be logged in for the before_request then those end points can never be reached